### ISSUE TITLE
[BUG]: Hover states never show for keyboard users

### ISSUE THREAD

**Issue by emmalearnscode** [JayantGoel001/JayantGoel001.github.io#17]
### Description of the bug

The hover states are inaccessible to keyboard users. Need to add the flip even on focus. They are also missing focus outlines to show which item is focussed

### Steps To Reproduce

Navigate through the site to the different sections. None of the cards flip to show what's on the back of the card.
